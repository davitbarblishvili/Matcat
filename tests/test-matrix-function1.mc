func m() matrix {
        return [[3, 4]];
}

func main() int {
    matrix m;
    m = m();
    printm(m);

    return 0;
}
