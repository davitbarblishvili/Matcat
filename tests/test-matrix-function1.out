(
[3.00 4.00 ]
)